docs/js_sqlite/multiplatform.md

```diff
@@ -0,0 +1,82 @@
+# Multiplatform setup with the SqlJs Driver
+
+{% include 'common/index_gradle_database.md' %}
+
+{% include 'common/index_schema.md' %}
+
+```groovy
+kotlin {
+  // The drivers needed will change depending on what platforms you target:
+
+  sourceSets.androidMain.dependencies {
+    implementation "com.squareup.sqldelight:android-driver:{{ versions.sqldelight }}"
+  }
+
+  // or sourceSets.iosMain, sourceSets.windowsMain, etc.
+  sourceSets.nativeMain.dependencies {
+    implementation "com.squareup.sqldelight:native-driver:{{ versions.sqldelight }}"
+  }
+
+  sourceSets.jvmMain.dependencies {
+    implementation "com.squareup.sqldelight:sqlite-driver:{{ versions.sqldelight }}"
+  }
+
+  sourceSets.jsMain.dependencies {
+    implementation "com.squareup.sqldelight:sqljs-driver:{{ versions.sqldelight }}"
+  }
+}
+```
+
+Because the SqlJs driver must be initialized asynchronously, the drivers for other platforms must be initialized in a compatible way to be usable in a common source set.
+
+The drivers can be initialized in a coroutine, and a higher-order function can be used to ensure that the driver is initialized before executing a block of code that requires the database:
+
+```kotlin
+// in src/commonMain/kotlin
+expect suspend fun provideDbDriver(schema: SqlDriver.Schema): SqlDriver
```

Comment:
Can't remember if we talked about this, but this is _exactly_ what I would build so 👍